src/methods.js

```diff
@@ -596,7 +596,17 @@ export default {
         named: params => set(params, 'privkeys', map(params.privkeys || [], () => '******'))
       }
     },
-    version: '>=0.7.0'
+    version: '>=0.7.0 <0.18.0'
+  },
+  signRawTransactionWithWallet: {
```

Comment:
Since we are deprecating `signRawTransaction`, please take in care also adding [signRawTransactionWithKey](https://github.com/bitcoin/bitcoin/blob/b5c3d7affc9890f13dbd2d13967238f798d984a3/src/rpc/rawtransaction.cpp) to allow to sign a transaction by passing private keys as argument.

Reply:
Also, sort the methods.